appjs/templates/project.ejs

```diff
@@ -77,22 +91,110 @@
         <div class="col-sm-8">
       <% } %>
         <div id="projectForm"></div>
+        <!-- how is it pulling in the project buttons here? -->
```

Comment:
Buttons are added in the postRender callback when the alpaca form is setup. Please remove this comment